File: make.lua
if string.find(string.lower(platformInfo()), "unix") then
    os.execute("cmake .")
    os.execute("make")
elseif string.find(string.lower(platformInfo()), "windows") then
    os.execute([[cmake -G "MinGW Makefiles"]])
    os.execute("mingw32-make")
end

print("make ok")

File: make.py
import os
import sys
import shutil
import platform

def make():
    ret = -1
    if platform.system().lower().find('windows') >= 0:
        os.system('cmake -G "MinGW Makefiles"')
        ret = os.system('mingw32-make')

    else:
        os.system("cmake .")
        ret = os.system("make")

    return ret

if __name__ == "__main__":
    sys.exit(make())
